File: Apps/Java/clientJava/src/main/java/com/mycompany/projeto/artefato/sprint/entidades/Staff.java
package com.mycompany.projeto.artefato.sprint.entidades;

public class Staff {
    private Integer id_staff;
    private String nome;
    private String email;
    private String senha;
    private String token;

    public Staff(Integer id_staff, String nome, String email, String senha,
            String token) {
        this.id_staff = id_staff;
        this.nome = nome;
        this.email = email;
        this.senha = senha;
        this.token = token;
    }
            
}

File: website/models/staff.js
'use strict';

module.exports = (sequelize, DataTypes) => {
    let staff = sequelize.define('staff',{	
		id: {
			field: 'id_staff',
			type: DataTypes.INTEGER,
			primaryKey: true,
			autoIncrement: true
		},	
		nome: {
			field: 'nome',
			type: DataTypes.STRING,
			allowNull: false
		},
		email: {
			field: 'email',
			type: DataTypes.STRING,
			allowNull: false
		},
		senha: {
			field: 'senha',
			type: DataTypes.STRING, 
			allowNull: false
		},
	}, 
	{
		tableName: 'staff', 
		freezeTableName: true, 
		underscored: true,
		timestamps: false,
	});

    return staff;
};
